skyVegasTests/utilities/saferGamblingHelper.ts

```diff
@@ -0,0 +1,6 @@
+import { type Page } from "@playwright/test";
+
+export let saferGamblingPage = async (sgPage:Page) => {
```

Comment:
perhaps rename this to navigateToSaferGamblingPage to have an action as function name?

Reply:
I am not sure we even need this file anymore as we don't go to the safer gambling page, due to the fact the account we use already has the reality check set

Reply:
Remove it then?